pgxn/neon/unstable_extensions.c

```diff
@@ -65,8 +65,8 @@ CheckUnstableExtension(
 			{
 				ereport(ERROR,
 						(errcode(ERRCODE_INSUFFICIENT_PRIVILEGE),
-						 errmsg("installing %s is currently prohibited", stmt->extname),
-						 errhint("Set neon.allow_unstable_extensions to true")));
+						 errmsg("%s extension is in beta and may be unstable or introduce backward-incompatible changes.\nWe recommend testing it in a separate, dedicated Neon project.", stmt->extname),
```

Comment:
I think it is best to steer clear of the term "beta." Why not also mention alpha? I think unstable covers this concept well.

Reply:
If you don't mind, I will keep current version to move forward with the release.
feel free to improve this later